mlos_core/mlos_core/tests/spaces/adapters/llamatune_test.py

```diff
@@ -238,25 +238,25 @@ def test_special_parameter_values_biasing():    # pylint: disable=too-complex
     adapter = LlamaTuneAdapter(input_space, num_low_dims=1, special_param_values=spv_dict,
                                max_unique_values_per_param=None)
 
-    special_values_occurences = {
+    special_values_instances = {
```

Comment:
In this case we rename the variable to also avoid a mypy warning about redefinition with different types.